packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -0,0 +1,479 @@
+          <Accordion className={styles['PricingOptions__feature-list-accordion']} open={false}>
+            <Accordion.Heading
+              as="h4"
+              className={styles['PricingOptions__feature-list-accordion-heading']}
+              reversedToggles
+            >
+              <ChevronDownIcon className={styles['PricingOptions__feature-list-accordion-chevron']} />
+              What&apos;s included
+            </Accordion.Heading>
+            <Accordion.Content className={styles['PricingOptions__feature-list-accordion-content']}>
+              {FeautreListItems}
+            </Accordion.Content>
+          </Accordion>
+        ) : (
+          <>
+            <Text size="200" variant="muted" className={styles['PricingOptions__feature-list-toggle']}>
+              What&apos;s included:
+            </Text>
+            {FeautreListItems}
+          </>
+        )}
+      </div>
+    )
+  },
+)
+
+type PricingOptionsFeatureHeadingProps = PropsWithChildren<BaseProps<HTMLHeadingElement>> & {
+  'data-testid'?: string
+}
+
+const PricingOptionsFeatureHeading = forwardRef<HTMLHeadingElement, PricingOptionsFeatureHeadingProps>(
+  ({children, className, 'data-testid': testId, ...rest}, ref) => {
+    return (
+      <Heading
+        as="h4"
+        className={clsx(styles['PricingOptions__feature-list-heading'], className)}
+        data-testid={testId || testIds.featureListHeading}
+        ref={ref}
+        size="subhead-medium"
+        {...rest}
+      >
+        {children}
+      </Heading>
+    )
+  },
+)
+
+type PricingOptionsFeatureListItemProps = PropsWithChildren<BaseProps<HTMLLIElement>> & {
+  'data-testid'?: string
+  variant?: 'included' | 'excluded'
+}
+
+const PricingOptionsFeatureListItem = forwardRef<HTMLLIElement, PricingOptionsFeatureListItemProps>(
+  ({children, className, variant = 'included', 'data-testid': testId, ...rest}, ref) => {
+    const itemLeadingVisual = variant === 'included' ? CheckIcon : XIcon
+    const itemLeadingVisualAriaLabel = variant === 'included' ? 'Includes' : 'Does not include'
+    const itemLeadingVisualFill =
+      variant === 'included' ? 'var(--brand-color-accent-primary)' : 'var(--brand-color-text-muted'
+
+    return (
+      <UnorderedList.Item
+        className={clsx(
+          styles['PricingOptions__feature-list-item'],
+          {[styles['PricingOptions__feature-list-item--excluded']]: variant === 'excluded'},
+          className,
+        )}
+        data-testid={testId || testIds.featureListItem}
+        leadingVisual={itemLeadingVisual}
+        leadingVisualAriaLabel={itemLeadingVisualAriaLabel}
+        leadingVisualFill={itemLeadingVisualFill}
+        ref={ref}
+        {...rest}
+      >
+        {children}
+      </UnorderedList.Item>
+    )
+  },
+)
+
+type RestrictedPolymorphism =
+  | (BaseProps<HTMLAnchorElement> & {as?: 'a'})
+  | (BaseProps<HTMLButtonElement> & {as?: 'button'})
+
+type PricingOptionsActionsProps = {
+  as?: 'a' | 'button'
```

Comment:
Do these need to be buttons too? If so, I think we need to use a discriminated union type here because `href` is a required prop, but doesn't apply to `button` elements.